weblate.md

```diff
@@ -1,83 +1,85 @@
 ## Step by step guide for Weblate
 
-Translators can make two actions on weblate: translate new strings or Review other contributor's translations. This guide will give all the info you need to perform both tasks. 
+Translators can perform two actions on Weblate: translate new strings or review the translations of others.
+This guide will give all the info you need to perform both tasks. 
 
 ### 0. Start!
-Go to the [project page](https://translate.getmonero.org/projects/monero/),click on the tool you would like to contribute to (CLI or GUI wallet) and select the language. You will see some statistics and a series of lines in different colours:
+Go to the [project page](https://translate.getmonero.org/projects/monero/), click on the tool you would like to contribute to (for example CLI or GUI wallet) and select the language. You will see some statistics and a series of lines in different colours:
 ![screenshot](/media/weblate/strings_status.png)
 
-**Green:** You can ignore these, unless you would like to review already translated strings.  
-**Red:** Untranslated srings or with failing checks. These should be the priority.  
-**grey:** Strings that need some kind of action.  
```

Comment:
The two trailing space at the end are necessary to show the strings correctly and not one after another.